libs/shared/src/lib/sass/mixins/connotation/_mixins.scss

```
@@ -36,3 +36,22 @@
 		);
 	}
 }
+
+@mixin variables-backwards-compatibility($component, $element-class) {
```

Let's add some comment to our crazy code

```suggestion
/**
 * Ghost text color has changed from primary to firm in VIV-1774. For backwards compatibility with code
 * setting primary we need to default firm to primary value if it is provided.
 */
@mixin variables-backwards-compatibility($component, $element-class) {
```